### PR TITLE
[FIX] web: fix notification design

### PR DESCRIPTION
**Pull request by jeh-odoo**
# Purpose 

Fix the design of the notification which is sometimes broken
(if there are lists or buttons display in it for example)

task-2346462

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
Not sure who to ping for this small fix @seb-odoo @aab-odoo

**Comment by seb-odoo** [comment]
Not me. I mean I could have a look but I don't have time today and it's only `web` code anyway.

File: addons/calendar/static/src/scss/calendar.scss [hunk 1]
```diff
@@ -67,3 +67,7 @@
         font-weight: bold;
     }
 }
+.o_notification_manager, .o_notification {
```
Comment (aab-odoo) [inline addons/calendar/static/src/scss/calendar.scss:70]: You probably don't want to do that: here you change the design for everyone. So either this is exactly what you want to do, and it should be in web, or you want to customize a calendar notif, and the rules should be properly scoped.

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**